update docs (#588)
kyma-project/btp-manager: docs/user/02-10-usage.md

     servicePlanName: application
     btpAccessCredentialsSecret: test-secret
   ```

> 5. To verify that ServiceInstance was created with success then run:

Reviewer: ```suggestion
5. To verify that the ServiceInstance has been created successfully, run:
```

    ```bash

>     kubectl get serviceinstances.services.cloud.sap.com test-service -o yaml

Reviewer: ```suggestion
    kubectl get serviceinstances.services.cloud.sap.com test-service-instance -o yaml
```

    ```

    You see the status `created` and the message `ServiceInstance provisioned successfully`.

>     Also you should see in one of spec field `btpAccessCredentialsSecret` the `test-secret` value.

Reviewer: ```suggestion
    You also see the `test-secret` value in the **btpAccessCredentialsSecret** spec field.
```

>     In status section you should also see not empty field `subaccountId`.

Reviewer: ```suggestion
    In the status section,  the **subaccount** field must not be empty.
```

6. Clean up your resources by running the following command:

    ```bash
    kubectl delete serviceinstances.services.cloud.sap.com test-service-instance
    ```

>     If you dont use `test-secret` for other ServiceInstances you can delete it by this command:

Reviewer: ```suggestion
    If you are not using the `test-secret` Secret for other ServiceInstances, you can delete it with this command:
```
